js/apps/account-ui/pom.xml

```diff
@@ -80,7 +90,9 @@
       "authUrl": "${authUrl}",
       "isRunningAsTheme": true,
       "realm": "${realm.name}",
-      "resourceUrl": "${resourceUrl}"
+      "resourceUrl": "${resourceUrl}",
+      "logo": "${properties.logo!""}",
+      "logoUrl": "${properties.logoUrl!""}"
```

Comment:
Is there a way that we can omit these properties entirely from the JSON if no default is provided? I think ideally we'd always fall back to the defaults described in `environment.ts`. If these values are defined those defaults will always get overwritten with empty strings.

Reply:
I could look into it, but I don't see why it matters.  And this way, it's self-documenting.  You can easily see what values are available.

Reply:
I could look into it, but I don't see why it matters.  And this way, it's self-documenting.  You can easily see what values are available.